opal/corelib/runtime.js

```diff
@@ -124,21 +124,139 @@
   //     Opal.Object.$$scope.get("Array")
   //
   // If a constant with the given name cannot be found, then a dispatch to the
-  // class/module's `#const_method` is called, which by default will raise an
+  // class/module's `#const_missing` is called, which by default will raise an
   // error.
   //
   // @param name [String] the name of the constant to lookup
   // @return [Object]
   //
-  Opal.get = function(name) {
-    var constant = this[name];
+  Opal.get = function(const_name) {
+    return Opal.const_get([this], const_name, true, true);
```

Comment:
This method can be used on any `$$scope`, i.e. :
``` js
Opal.get('String').$$scope.get('Wrapper').$name()
=> "String::Wrapper"
```
`this` is a `$$scope`, so we can't avoid it (and I guess we shouldn't)